### PR TITLE
Created ProtoInput

### PR DESCRIPTION
**Pull request by Nikolas023**
I created ProtoInput and added javadoc comments to it. This will get the user input. In ProtoUser, a compute request will be sent to the compute engine, which will determine if the input is valid or not using the AckToken.

### CODE REVIEW

File: .classpath
Comment (dgunther2001) [inline .classpath:1]: Add to gitignore

File: .project
Comment (dgunther2001) [inline .project:1]: Add to gitignore

File: .settings/org.eclipse.buildship.core.prefs
Comment (dgunther2001) [inline .settings/org.eclipse.buildship.core.prefs:1]: Add to gitignore

File: src/Engine/AckToken.java
Comment (dgunther2001) [inline src/Engine/AckToken.java:1]: Looks great!

File: src/Engine/ProtoUser.java
Comment (dgunther2001) [inline src/Engine/ProtoUser.java:1]: Checkstyle fails because it wants comment above the function

File: src/engine/ProtoInput.java
Comment (dgunther2001) [inline src/engine/ProtoInput.java:1]: Function needs javadoc to pass checkstyle

File: src/Engine/AckToken.java
Comment (dgunther2001) [reply src/Engine/AckToken.java:1]: Missing javadoc comments though (check checkstyle failure log for exact locations)

**Review by dgunther2001 — CHANGES_REQUESTED** [review]
This looks good, but please add apis in the compute initializer to receive user requests and also send a result back to the user. Please add config files into the gitignore as this will affect nico and I's eclipse config and potentially break it. Also, please check the actions logs for where javadoc comments are missing. The logic here is great, just a few things to check on and add.

**Comment by Nikolas023** [comment]
> This looks good, but please add apis in the compute initializer to receive user requests and also send a result back to the user. Please add config files into the gitignore as this will affect nico and I's eclipse config and potentially break it. Also, please check the actions logs for where javadoc comments are missing. The logic here is great, just a few things to check on and add.

Just updated everything!

**Review by dgunther2001 — APPROVED** [review]
LGTM. Consider adding a req and res function to the compute initializer as the api needs to work on both ends, not just in the user end.

**Pull request merged**